_posts/2016-7-4-this-week-in-rust-docs-11.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: This Week in Rust Docs 11
+date: 2016-07-04
+---
+
+Hello and welcome to *This Week in Rust Docs*!
+
+*This Week in Rust Docs* is openly developed [on GitHub](https://github.com/GuillaumeGomez/this-week-in-rust-docs).
+If you find any errors in this week's issue, [please submit a PR](https://github.com/GuillaumeGomez/this-week-in-rust-docs/pulls).
+
+And of course, don't forget to look at the docs:
+
+* [Stable](https://doc.rust-lang.org/)
+* [Beta](http://doc.rust-lang.org/beta/)
+* [Nightly](http://doc.rust-lang.org/nightly/)
+
+This week's edition was edited by: [GuillaumeGomez](https://github.com/GuillaumeGomez).
+
+# Latest news
+
+A [post](https://facility9.com/2016/07/rust-doc-days-follow-up/) has been wrote about the rust doc days, thanks to [@peschkaj](https://github.com/peschkaj)!
+
+# Current opened issues
+
+For now, here are the two big issues opened for Rust documentation:
+
+* [The Standard Library Documentation Checklist](https://github.com/rust-lang/rust/issues/29329)
+* [Add error explanations for all error codes](https://github.com/rust-lang/rust/issues/32777)
+
+They both need help to move forward so any contribution is very welcome!
+
+There are currently around 50 other documentation issues opened. Look for [A-docs](https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AA-docs) tagged issues on github!
+
+# Call for participation
+
+There's now a call for participation to [display all methods of a type, even those from implicit traits](https://github.com/rust-lang/rust/issues/33772). This is a great way to help users find everything that a type can do. Any help on it would be very appreciated!
```

Comment:
Could you mention that this is about extending rustdoc and not about writing docs? :)